libraries/chain/kv_context_rocksdb.cpp

```diff
@@ -213,16 +217,17 @@ namespace eosio { namespace chain {
                old_value = view.get(contract, { key, key_size });
                if (!old_value)
                   return 0;
+               EOS_ASSERT(old_value->size() >= kv_payer_size, kv_rocksdb_bad_value_size_exception , "The size of value returned from RocksDB is less than payer's size");
                view.erase(contract, { key, key_size });
             }
             FC_LOG_AND_RETHROW()
          }
          CATCH_AND_EXIT_DB_FAILURE()
 
-         int64_t resource_delta = -(static_cast<int64_t>(resource_manager.billable_size) + key_size + old_value->size());
-# warning TODO: Investigate where to store payer
-         //resource_manager.update_table_usage(resource_delta, kv_resource_trace(key, key_size, kv_resource_trace::operation::erase));
-         return resource_delta;
+         account_name payer;
```

Comment:
make a static method that takes a "value" and returns the payer, and use here and elsewhere

Reply:
would prefer for get_payer to just take the value data ptr and create the account_name in the function and return it, it is only a uint64_t and since it is returned from the function the compiler will move the memory anyways